config/components/prometheus/monitor.yaml

```diff
@@ -0,0 +1,16 @@
+
```

Comment:
nitpick: spacing, not sure if this is auto-gen or not. Feel free to ignore

Reply:
The files *were* autogenerated but I don't think anything will touch them going forward. I'll fix the weird spacing.